### DISCUSSION TITLE
Could you please take a look at my portfolio project developed with React JS and share your thoughts on it?

### DISCUSSION THREAD

**Discussion by detectivesoftware** [community/community#127438] [Programming Help]
### Select Topic Area

Question ***Could you please take a look at my portfolio project developed with React JS and share your thoughts on it?***

### Body

*Could you please take a look at my portfolio project developed with React JS and share your thoughts on it?**

_The github repository_
https://github.com/mz-dz/portfolio-ibra

_"This is the link."_
[https://ibra-alpha.vercel.app/](https://ibra-alpha.vercel.app/)

**Comment by LiteBrite82** [comment]
Hi @mz-dz 
I get a GitHub 404 error on each link. Is your repo set to private?

**Reply by detectivesoftware** [reply]
> Hi @mz-dz I get a GitHub 404 error on each link. Is your repo set to private?
---

No, it's a public repo. 

"Try again." **_@LiteBrite82_**

**Comment by detectivesoftware** [comment]
@LiteBrite82

**Comment by SubhaBhatta** [comment]
Your portfolio website is very amazing. The code base is great. Overally, your portfolio is good.

**Reply by detectivesoftware** [reply]
Thank you bro

**Comment by go2usman** [comment]
- Work on UX part
- add some interactive UI, like graphics, so that user can easily understand no need to Read the whole para its like a Blog not a portfolio to be honest
- your second link is not working 
[https://ibra-alpha.vercel.app/](https://github.com/orgs/community/discussions/url) this one
-  just correct it
`
[https://ibra-alpha.vercel.app/](https://ibra-alpha.vercel.app/) 
`
- Over All It is Okay

**Reply by detectivesoftware** [reply]
Thank you.

**Reply by detectivesoftware** [reply]
I want all my websites to have a consistent and cohesive style. (at the same style)
@github-usman

**Comment by YuheshPandian** [comment]
The UI is neat and minimal, this is just a starting point  which you could use to expand your skills.
Overall Rating:7/10. This is based on your experience and the skills you know for now.1

Thank you.

**Reply by detectivesoftware** [reply]
Thanks

**Comment by ebndev** [comment]
Hey @mz-dz, thanks for being a part of the GitHub Community!

Looks like you've gotten some good feedback so far 😄 

I agree with @github-usman & @YuheshPandian about this being a great starting point!

What are you going to use your portfolio for? Getting clients? Showcasing your work? I don't see any immediate call to actions or feel like I got a good sense of who you are as a developer / designer.

Your GitHub already shows all of the same information in the same format, so your portfolio should show more of your skills and personality. Definitely some images of your work would be helpful!

**Reply by detectivesoftware** [reply]
it's great 😌
